src/commands/deployAzurePipeline/browsePipeline.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { CloudExplorerV1 } from 'vscode-kubernetes-tools-api';
+import { AksClusterTreeNode } from '../../tree/aksClusterTreeItem';
+
+export async function browsePipeline(context: IActionContext, target: any): Promise<void> {
+    const deploymentCenterUrl = await getDeploymentCenterUrl(target);
```

Comment:
💡 :) since its no longer an `async`, this specific `await` is redundant. (you forgot this place) https://github.com/Azure/vscode-aks-tools/pull/42/commits/e162e3de51ec5d686421fdefc56c24655adf308b